frontend/src/metabase/entities/questions.js

```diff
@@ -109,15 +129,31 @@ const Questions = createEntity({
         undo(opts, getLabel(card), archived ? t`trashed` : t`restored`),
       ),
 
-    setCollection: (card, collection, opts) => {
+    // NOTE: standard questions (i.e. not models, metrics, etc.) can live in dashboards as well as collections.
+    // this function name is incorrectly but maintained for consistency with other entities.
+    setCollection: (card, destination, opts) => {
       return async dispatch => {
+        const archived =
+          destination.model === "collection" &&
+          isRootTrashCollection(destination);
+
+        const update =
+          destination.model === "dashboard"
+            ? {
+                dashboard_id: destination.id,
+                archived,
+                delete_old_dashcards: true,
```

Comment:
@npfitz I couldn't get this to work as a query param this way, did we move this to the request body?

Reply:
Yeah, it gets handled in `api/card.ts`. If it exists we pull it from the body and add it to the URL